events_task/lib/my_app/content/confirmation.ex

```diff
@@ -0,0 +1,17 @@
+defmodule MyApp.Content.Confirmation do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "confirmations" do
+    belongs_to(:user, User)
+    belongs_to(:event, Event)
+    timestamps()
+  end
+
+  def changeset(confirmation, attrs) do
+    confirmation
+    |> cast(attrs, [:user_id, :event_id])
+    |> unique_constraint(:user_event, name: :user_event)
```

Comment:
https://hexdocs.pm/ecto/Ecto.Changeset.html#unique_constraint/3
First argument should be field.